doc/neps/nep-0046-sponsorship-guidelines.rst

```diff
@@ -0,0 +1,254 @@
+.. _NEP46:
+
+=====================================
+NEP 46 — NumPy Sponsorship Guidelines
+=====================================
+
+:Author: Ralf Gommers <ralf.gommers@gmail.com>
+:Status: Draft
+:Type: Process
+:Created: 2020-12-27
+:Resolution: <url> (required for Accepted | Rejected | Withdrawn)
+
+
+Abstract
+--------
+
+This NEP provides guidelines on how the NumPy project will acknowledge
+financial and in-kind support.
+
+
+Motivation and Scope
+--------------------
+
+In the past few years the NumPy project has gotten significant financial
+support, as well as dedicated work time for maintainers to work on NumPy. There
+is a need to acknowledge that support - funders and organizations expect or require
+it, it's helpful when looking for new funding, and it's the right thing to do.
+Furthermore, having a clear policy for how NumPy acknowledges support is
+helpful when searching for new support.
+
+This NEP is aimed at both the NumPy community - who can use it when looking for
+support and acknowledging existing support - and at past, current and
+prospective sponsors, who often want or need to know what they get in return
+for their support other than a healthier NumPy.
+
+The scope of this proposal includes:
+
+- direct financial support, employers providing paid time for NumPy maintainers
+  and regular contributors, and in-kind support such as free hardware resources or
+  services.
+- where and how NumPy acknowledges support (e.g., logo placement on the website).
+- the amount and duration of support which leads to acknowledgement.
+- who in the NumPy project is responsible for sponsorship related topics, and
+  how to contact them.
+
+
+How NumPy will acknowledge support
+----------------------------------
+
+There will be two different ways to acknowledge financial and in-kind support,
+one to recognize significant active support, and another one to recognize
+support received in the past and smaller amounts of support.
+
+Entities who fall under "significant active supporter" we'll call Sponsor.
+The minimum level of support given to NumPy to be considered a Sponsor are:
+
+- $30,000/yr for unrestricted financial contributions
+- $60,000/yr for financial contributions for a particular purpose
+- $100,000/yr for in-kind contributions
+
+We define support being active as:
+
+- for a one-off donation: it was received within the previous 12 months
+- for a grant or other support with a defined start and end date: the duration
```

Comment:
This is the first time that "grant" is mentioned in this document.  Grants do not typically go to NumPy, so they are in-kind contributions of time, I imagine.  Perhaps, then, this should be: if you are actively committing in-kind contributions on the order of 100k/yr you are recognized.

Reply:
They can go to NumPy, see the two CZI grants. And even if other grants technically go to a PI at an institute, I'd still say those grants "go to the projects". It's not academic research, where it's the PI and the idea that are primary - the intention of all four grants we've had so far was to support NumPy.

I'm happy to make a tweak to the wording though, pushing a change similar to what you propose.

Note that I'd put grants under "financial contributions for a particular purpose" not "in-kind contributions". So it'd be $60k/yr.

Reply:
If you could clarify that last point, that would be good.  I consider the NumPy grant at BIDS a grant to NumPy as well, but it starts to get tricky because of the overheads going to the university etc.  So how much actually ends up at NumPy?  That is why I prefer to measure our contribution in person months given, because that, realistically, is what we can do with the money.

Reply:
Yes, which we do - see Institutional Partners. So Berkeley is a partner, >= 6 person-months. Moore/Sloan are sponsors, >$60k/yr.